<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiCameraSetupClass</class>
 <widget class="QWidget" name="QtGuiCameraSetupClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtGuiCameraSetup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_15">
   <item>
    <widget class="QTabWidget" name="setupSensorParams">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_brightless">
      <attribute name="title">
       <string>Brightless</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="GB_exprosure">
         <property name="title">
          <string>Exposure</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="4,1,2">
            <item>
             <widget class="QSlider" name="horSlider_exprosureSeconds">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>60</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>20</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_exprosureSeconds">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>+ seconds</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="4,1,2">
            <item>
             <widget class="QSlider" name="horSlider_exprosureMilSeconds">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_exprosureMilSeconds">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>+ mili seconds</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="4,1,2">
            <item>
             <widget class="QSlider" name="horSlider_exprosureMicSeconds">
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_exprosureMicSecods">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>+ micro seconds</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="2,1,3,1,2">
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>=</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_exprosureSumm">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>micro seconds</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QPushButton" name="pb_continuous">
              <property name="text">
               <string>Continuous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_once">
              <property name="text">
               <string>Once</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_off">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Off</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GroupBox_gain">
         <property name="title">
          <string>Gain</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="4,1,2">
            <item>
             <widget class="QSlider" name="horSlider_gain">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>32</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_gain">
              <property name="text">
               <string>0</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>db</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19">
            <item>
             <widget class="QPushButton" name="pb_GainContinuous">
              <property name="text">
               <string>Continuous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_GainOonce">
              <property name="text">
               <string>Once</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_GainOff">
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Off</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoRepeat">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GroupBox_exposureAutoTarget">
         <property name="title">
          <string>Exprosure auto target</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="4,1,2">
            <item>
             <widget class="QSlider" name="horSlider_exprosureAutoTarget">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>50</number>
              </property>
              <property name="tracking">
               <bool>true</bool>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
              <property name="tickPosition">
               <enum>QSlider::TicksAbove</enum>
              </property>
              <property name="tickInterval">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_exprosureAutoTarget">
              <property name="text">
               <string>50</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_roi">
      <attribute name="title">
       <string>ROI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Image Format</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Pixel Format:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QComboBox" name="comboBox_pixelFormat">
            <item>
             <property name="text">
              <string>Mono8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mono12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mono12Packed</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Binning Horizontal:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="spinBox_binningHor">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>8</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Binning Vertical:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QSpinBox" name="spinBox_binningVer">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>11</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>67</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>ROI</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <spacer name="horizontalSpacer_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="Widget" name="setRoiWid" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>256</width>
                <height>256</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>256</width>
                <height>256</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_15">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>ROI coordinate</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2" columnstretch="2,1,2,1">
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_height">
               <property name="maximum">
                <number>3248</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>OffsetY</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSpinBox" name="spinBox_ofsetY">
               <property name="maximum">
                <number>3248</number>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QSpinBox" name="spinBox_ofsetX">
               <property name="maximum">
                <number>4872</number>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Width</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>OffsetX</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="spinBox_width">
               <property name="maximum">
                <number>4872</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QPushButton" name="pb_full">
              <property name="text">
               <string>Full</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_oneQuarter">
              <property name="text">
               <string>1/4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_oneEighth">
              <property name="text">
               <string>1/8</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23" stretch="1,1,1">
            <item>
             <spacer name="horizontalSpacer_16">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pb_setRoi">
              <property name="text">
               <string>Set ROI</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_17">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header>widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
